packages/react-liveness/src/components/FaceLivenessDetector/FaceLivenessDetector.tsx

```diff
@@ -29,70 +18,17 @@ export interface FaceLivenessDetectorProps
 export default function FaceLivenessDetector(
   props: FaceLivenessDetectorProps
 ): JSX.Element {
-  const {
-    disableInstructionScreen = false,
-    components,
-    config,
-    displayText,
-  } = props;
-  const currElementRef = React.useRef<HTMLDivElement>(null);
-  const {
-    hintDisplayText,
-    cameraDisplayText,
-    instructionDisplayText,
-    streamDisplayText,
-    errorDisplayText,
-  } = getDisplayText(displayText);
-
-  const service = useInterpret(livenessMachine, {
-    devTools: process.env.NODE_ENV === 'development',
-    context: {
-      componentProps: {
-        ...props,
-        config: config ?? {},
-      },
-    },
-  });
-
-  const [state, send] = useActor(service);
-  const isStartView = state.matches('start') || state.matches('userCancel');
-
-  const beginLivenessCheck = React.useCallback(() => {
-    const videoConstraints = getVideoConstraints();
-
-    send({
-      type: 'BEGIN',
-      data: { videoConstraints },
-    });
-  }, [send]);
-
-  React.useLayoutEffect(() => {
-    if (disableInstructionScreen && isStartView) {
-      beginLivenessCheck();
-    }
-  }, [beginLivenessCheck, disableInstructionScreen, isStartView]);
+  const amplifyCredentialsProvider = async () => {
+    const credentials =
+      (await AmplifyCredentials.get()) as AwsTemporaryCredentials;
+    return credentials;
+  };
 
+  const { config } = props;
```

Comment:
Prefer destructuring `...rest` as well here and passing that `FaceLivenessDetectorCore` to explicitly prevent `config` from being passed twice, example:
```
const { config, ...rest } = props;
return (
  <FaceLivenessDetectorCore
    {...rest}
    config={{ credentialProvider: amplifyCredentialsProvider, ...config }}
  />
)
```